inventory/sample/group_vars/k8s_cluster/k8s-cluster.yml

```diff
@@ -315,8 +315,9 @@ persistent_volumes_enabled: false
 ## Important: if you use Ubuntu then you should set in all.yml 'docker_storage_options: -s overlay2'
 ## Array with nvida_gpu_nodes, leave empty or comment if you don't want to install drivers.
 ## Labels and taints won't be set to nodes if they are not in the array.
-# nvidia_gpu_nodes:
-#   - kube-gpu-001
+## Dynamic render gpu node list via ansible inventory group.
+# nvidia_gpu_nodes: "{{ groups['nvidia_gpu_nodes'] | list }}"
+
```

Comment:
You can drop this, the nvidia_gpu_nodes which only reflect the groups is a bad idea and will be deleted eventually.